packages/web-app-files/src/components/SideBar/Shares/Collaborators/EditDropdown.vue

```diff
@@ -186,10 +190,7 @@ export default defineComponent({
     },
 
     isExpirationSupported() {
-      return (
-        (this.editingUser && this.userExpirationDate) ||
-        (this.editingGroup && this.groupExpirationDate)
```

Comment:
the expiration capability is just for default expiration dates (bad naming tough)